builtin/game/chatcommands.lua

```diff
@@ -985,3 +985,40 @@ core.register_chatcommand("clearinv", {
 		end
 	end,
 })
+
+local function handle_kill_command(murderer, victim)
+	if core.settings:get_bool("enable_damage") == false then
+		return false, "Players can't be killed right now, damage has been disabled."
```

Comment:
This map-wide setting is not meant to be changed in-game -> `Players cannot be killed, damage has been disabled.`